README.md

```diff
@@ -1,65 +1,24 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse
 
-> One paragraph statement about the project.
+> This project is only displaying Hello Microverse.
```

Comment:
- [ ] Please make sure that your README contains the steps for helping users set up a local copy of your project.

For example:
```
## Setup 
#### To get a local copy up and running follow these simple example steps.
- Click the green Code button on the repo
- In your local PC, open your terminal in the folder you would like to clone the project.
- ...
- ...
- and so on
```